### PR TITLE
Adds webpage for javascript injector example

### PR DESCRIPTION
**Pull request by iamplugged**
- Adds Hello World webpage
- Adds text replacement script
- Modifies GH actions

To Do:
- Documentation for JS injection

image

### CODE REVIEW

File: .github/workflows/deploy-js-hello-world.yml [hunk 1]
```diff
@@ -31,15 +31,12 @@ jobs:
...
-        uses: actions/jekyll-build-pages@v1
-        with:
-          source: ./javascript-injectors/hello-world/pages
-          destination: ./_site/hello-world
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
+      - name: Build Pages
+        working-directory: ./javascript-injectors/hello-world/pages
+        run: | 
+          chmod +x ./build.sh
```
Comment (vpetersson) [inline .github/workflows/deploy-js-hello-world.yml:37]: Feels a bit overkill with a script with just two lines. Just run it here instead as it makes it easier to debug.

File: javascript-injectors/hello-world/pages/index.html [hunk 2]
```diff
@@ -14,31 +14,26 @@
         <link href="https://fonts.googleapis.com/css?family=Inter" rel="stylesheet">
         <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet">
 
-        <link href="../../theme/playground-theme.css" rel="stylesheet">
-        <link href="./assets/styles/main.css" rel="stylesheet">
+        <link href="../../../bootstrap/theme/playground-theme.css" rel="stylesheet">
```
Comment (vpetersson) [inline javascript-injectors/hello-world/pages/index.html:17]: This doesn't look right to me. If we are copying in the `boostrap` folder in the build process, why do we need a relative path?

Comment (iamplugged) [reply javascript-injectors/hello-world/pages/index.html:17]: @vpetersson This is for dev. While developing locally we just reference the original source.

File: .github/workflows/deploy-js-hello-world.yml (hunk 1 above)
Comment (iamplugged) [reply .github/workflows/deploy-js-hello-world.yml]: Made the change.

File: javascript-injectors/hello-world/pages/index.html (hunk 2 above)
Comment (vpetersson) [reply javascript-injectors/hello-world/pages/index.html:17]: ah right - hence the `sed`. It's fine for now but feels a bit hackish.

**Review by vpetersson — APPROVED** [review]

**Pull request merged**